lib/turkish_cities.rb

```diff
@@ -6,7 +6,22 @@
 require_relative '../lib/turkish_cities/postcode'
 require_relative '../lib/turkish_cities/version'
 
+require 'i18n'
+
+I18n.load_path << Dir["#{File.expand_path('config/locales')}/*.yml"]
+
 class TurkishCities
+  PLATE_NUMBER = 'plate_number'
+  PHONE_CODE = 'phone_code'
+
+  def self.change_locale(language_code)
```

Comment:
Not crucial but adding a test for this method as well would be beneficial for coverage.